README.md

```diff
@@ -39,10 +38,7 @@ keyring, or the keyring does not contain the relevant key, you will be prompted
 the password. The password will only be used to get a token from the Empower Web API.
 When the token runs out, you will have to input your password again.
 
-This isn't the best way to use EmpowerHandler, since it is easy to forget to log out,
-which can negatively impact the API server. Therefor, you need to tell that you want to
-use it this way, and you will still get a warning. When you are done developing your
-application, you should only login from a context manager:
+To log in, use the EMpowerHandler with a context manager:
```

Comment:
Typo

Reply:
Fixed (and added code styling to every mention)